### PR TITLE
[Modular] Ghost Role Bank Accounts + freighter/BMD loadout 

### PR DESCRIPTION
**Pull request by Zenitheevee**
## About The Pull Request

DS-2, Interdyne, Tarkon, Freighter, Hotel and Black Market Dealer now have Bank Accounts, including memories for the account ID should they lose it.

Black Market Dealer and Freighter have Loadout (And quirks for freighter) Enabled

## How This Contributes To The Skyrat Roleplay Experience

[I mean we had centralized banking for years.](https://www.youtube.com/watch?v=Y16P3I0doEg)

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
bankies
bmdbank
hotelbank
syndiebank
tarkonbank
image
image

</details>

## Changelog

:cl:
qol: Most ghost roles now have bank accounts + their account ID in memories
qol: Freighter + BMD now have loadout enabled
/:cl:

### CODE REVIEW

**Comment by Zergspower** [comment]
i blinked for a moment when i saw BDM wondering why they needed an account as an NPC lol

**Comment by Zenitheevee** [comment]
Hold up for a minute on this, I'm throwing errors atm on playing in testing.

**Comment by Zenitheevee** [comment]
> i blinked for a moment when i saw BDM wondering why they needed an account as an NPC lol

come on he's worked hard for all that money. Especially _his_ line of work.

**Comment by Zergspower** [comment]
(Though funny aside the CL should be fixed to BMD)

**Comment by Zenitheevee** [comment]
Also something to keep in mind: This system (for applying bank account) Is going to get a bit of a rework when i eventually go into giving ghost roles actual job datums. 

Like, I'm not sure this is going to go as far as alternate ghost role names but the most ambitious i'm aiming is ghost role department banks

File: modular_skyrat/modules/mapping/code/mob_spawns.dm [hunk 1]
```diff
@@ -341,6 +348,15 @@
 	back = /obj/item/storage/backpack
 	id = /obj/item/card/id/away/freightcrew
 
+/datum/outfit/freighter_crew/post_equip(mob/living/carbon/human/crewman, visualsOnly)
+	var/obj/item/card/id/id_card = crewman.wear_id
+	if(istype(id_card))
+		id_card.registered_name = crewman.real_name
+		id_card.update_label()
+		id_card.update_icon()
+	handlebank(crewman)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/mob_spawns.dm:360]: For all of these, please just put a space before the `handlebank()` call please <3
```suggestion

	handlebank(crewman)
```

File: modular_skyrat/modules/mapping/code/mob_spawns.dm [hunk 2]
```diff
@@ -370,6 +388,15 @@
 	r_pocket = /obj/item/storage/bag/ore
 	id = /obj/item/card/id/away/freightmine
 
+/datum/outfit/freighter_excavator/post_equip(mob/living/carbon/human/crewman, visualsOnly)
+	var/obj/item/card/id/id_card = crewman.wear_id
+	if(istype(id_card))
+		id_card.registered_name = crewman.real_name
+		id_card.update_label()
+		id_card.update_icon()
+	handlebank(crewman)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/mob_spawns.dm:401]:
```suggestion

	handlebank(crewman)
```

File: modular_skyrat/modules/mapping/code/mob_spawns.dm [hunk 3]
```diff
@@ -302,7 +307,7 @@
 		id_card.registered_name = staff.real_name
 		id_card.update_label()
 		id_card.update_icon()
-
+	handlebank(staff)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/mob_spawns.dm:312]:
```suggestion

	handlebank(staff)
```

File: modular_skyrat/modules/mapping/code/mob_spawns.dm [hunk 4]
```diff
@@ -138,7 +143,7 @@
 		id_card.registered_name = syndicate.real_name
 		id_card.update_label()
 		id_card.update_icon()
-
+	handlebank(syndicate)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/code/mob_spawns.dm:147]:
```suggestion

	handlebank(syndicate)
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Good change though.

**Review by GoldenAlpharex — APPROVED** [review]
Should be good to go, thank you!

**Pull request merged**